test/autowaiting.spec.js

```diff
@@ -32,7 +32,7 @@ describe('Auto waiting', () => {
       page.click('a').then(() => messages.push('click')),
       page.waitForNavigation({ waitUntil: 'domcontentloaded' }).then(() => messages.push('domcontentloaded')),
     ]);
-    expect(messages.join('|')).toBe('route|domcontentloaded|click');
+    expect(messages.join('|')).toBe('route|click|domcontentloaded');
```

Comment:
Let's change these tests to log 'commit' using the 'framenavigated' event. In the current form they are not testing anything.